docs/management/managing-indices.asciidoc

```diff
@@ -2,32 +2,45 @@
 [[managing-indices]]
 == Index Management
 
-*Index Management* enables you to view index settings,
-mappings, and statistics and perform index-level operations.
-These include refreshing, flushing, clearing the cache, force merging segments,
-freezing indices, and more. Practicing good index management helps ensure
-that your data is stored in the most cost-effective way possible.
+[float]
+=== What you'll learn
```

Comment:
There should be an intro above "What you'll learn". It should describe "What it is and why you should use it"